index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="index.css" />
+</head>
+<body>
+    <main>
+        <header class="header">
+            <a href="#">My Logo</a>
```

Comment:
- [ ] Good job implementing changes required by the previous reviewer. But I noticed that the logo text doesn't match the Figma design. Your logo text has underline decoration that is not on the Figma template. You can fix this by only adding this code to your `style.css` file: `a { text-decoration: none; }`